### ISSUE TITLE
I encountered a problem using my own generated MWM file via Maps_generator

### ISSUE THREAD

**Issue by gitdhs** [mapsme/omim#14177]
Can't add map France_Ile-de-France_Paris ( UnsupportedFileFormat ). Probably it's already added or has newer data version.

**Comment by gitdhs** [comment]
please look https://github.com/organicmaps/organicmaps/issues/1115

**Comment by gitdhs** [comment]
I found the problem. The version of the customized mwm file is v11, but the latest version that can be parsed in platform/mwm_version.h is v10. How can I solve this problem?Thanks

**Comment by gitdhs** [comment]
@tatiana-yan .Sorry，I saw you were the last person to modify the platform/mwm_version.h. Can you give me some suggestions, or how can I make the MWM version become V10 / V9 via maps_generator, @maksimandrianov
thanks

**Comment by gitdhs** [comment]
Another question,when i add osm node-id  that exceeds the int type and is long int, I cannot generate an MWM file,Can the long int type be supported by modifying the maps_generator script?
Thanks!!!

**Comment by tatiana-yan** [comment]
maps_generator builds the latest mwm version. To build mwms for old release you need to checkout corresponding release branch and use generator_tool from it.
See readme, step 0: https://github.com/mapsme/omim/blob/master/tools/python/maps_generator/README.md

**Comment by gitdhs** [comment]
> maps_generator 构建最新的 mwm 版本。要为旧版本构建 mwms，您需要出各自的版本并测试使用 generator_tool。
> 请参阅自述文件，第 0 步：[https](https://github.com/mapsme/omim/blob/master/tools/python/maps_generator/README.md) : [//github.com/mapsme/omim/blob/ master/tools/python/maps_generator/README.md](https://github.com/mapsme/omim/blob/master/tools/python/maps_generator/README.md)

Yes, I'm using matching maps_generator generates MWM files. Can I extend the size of node ID? I want to use long int type
